setup.py

```diff
@@ -0,0 +1,48 @@
+import setuptools
+
+with open("readme.md", "r") as f:
+    long_description = f.read()
+
+setuptools.setup(
+    name="open-mcr",
+    url = "https://github.com/gutow/open-mcr",
```

Comment:
We should make this the URL of the original repo:
```suggestion
    url = "https://github.com/iansan5653/open-mcr",
```